2017/5/day5.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+)
+
+func main() {
+	file, err := os.Open("input.txt")
+	if err != nil {
+		log.Fatalf("Couldn't open file: %v\n", err)
+	}
+	defer file.Close()
+
+	var instr []int
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		instrStr := scanner.Text()
+		i, err := strconv.Atoi(instrStr)
```

Comment:
@BlueMonday do you know of any cleverer / shorter ways to parse ints from a file than this?

Reply:
I believe this is one of the better ways.

Reply:
kk :D i just have to get used to a bit more repetition i think